content/docs/tutorials/azure/azure-ts-aks-mean.md

```diff
@@ -32,22 +32,9 @@ We will be deploying to Azure, so you will need an Azure account. If you don't h
 [sign up for free here](https://azure.microsoft.com/en-us/free/). [Follow the instructions
 here](https://www.pulumi.com/docs/intro/cloud-providers/azure/setup/) to connect Pulumi to your Azure account.
 
-This example deploys a Helm Chart from [Bitnami's Helm chart
-repository](https://github.com/bitnami/charts), so you will need to [install the Helm
-CLI](https://docs.helm.sh/using_helm/#installing-helm) and configure it:
```

Comment:
Is the Helm CLI install step no longer needed?

Reply:
I guess not! Looks like this was removed as part of https://github.com/pulumi/examples/commit/5c20fbd4bd6df6d0edf43beda61a2f65a19f9bf0.